superdesk/io/feed_parsers/__init__.py

```diff
@@ -63,13 +63,15 @@ def set_dateline(self, item, city=None, text=None):
         :type text: str
         """
 
-        item['dateline'] = {}
+        item.setdefault('dateline', {})
 
         if city:
             cities = find_cities()
             located = [c for c in cities if c['city'] == city]
             item['dateline']['located'] = located[0] if len(located) > 0 else {'city_code': city, 'city': city,
                                                                                'tz': 'UTC', 'dateline': 'city'}
+            item['dateline'].setdefault('text', city)
```

Comment:
Problem is what date component and source to use?

I will remove the attempt to set the dateline altogether. It seems to be always Bangalore any way!